tests/FunctionsTest.php

```diff
@@ -185,6 +191,65 @@ public static function captureLastErrorDataProvider(): \Generator
         ];
     }
 
+    /**
+     * @dataProvider captureCheckInProvider
+     */
+    public function testCaptureCheckIn(array $expectedFunctionCallArgs)
+    {
+        $hub = new Hub();
+        $options = new Options([
+            'environment' => Event::DEFAULT_ENVIRONMENT,
+            'release' => '1.1.8',
+        ]);
+        /** @var ClientInterface&MockObject $client */
+        $client = $this->createMock(ClientInterface::class);
+        $client->expects($this->once())
+            ->method('getOptions')
+            ->willReturn($options);
+
+        SentrySdk::setCurrentHub($hub);
+
+        $this->assertNull(captureCheckIn(...$expectedFunctionCallArgs));
+        $hub->bindClient($client);
+
+        $checkIn = captureCheckIn(...$expectedFunctionCallArgs);
+        $this->assertInstanceOf(CheckIn::class, $checkIn);
+        $this->assertSame(Event::DEFAULT_ENVIRONMENT, $checkIn->getEnvironment());
+        $this->assertSame('1.1.8', $checkIn->getRelease());
+    }
+
+    public static function captureCheckInProvider(): \Generator
```

Comment:
I'm not sure I understand why this data provider is needed: the only difference that I see is that one test case returns a monitor for a crontab schedule while the other returns a monitor for an interval schedule. However, I fail to see how this could affect the public API: at this level of unit testing, we should only care about the fact that the call is proxied to the hub instance. This comment applies to the rest of the tests as well.